src/cromshell/metadata/command.py

```diff
@@ -50,7 +50,7 @@ def check_cromwell_server(config, workflow_id):
 
 
 def format_metadata_params(
-    list_of_keys: list, exclude_keys: bool, expand_subworkflow: bool
+    list_of_keys: list, exclude_keys: bool, dont_expand_subworkflows: bool
```

Comment:
So I would leave this as expand_subworkflow and invert the argument when you call it.  It's kind of confusing to deal with he conditions below that require multiple negations.  'if dont_expand == false then" is much easier to confuses than "if expand = true".

My reasoning when dealing with stuff like this is basically for input options, make the option correspond to what the person is setting the flag to do.  If the default is to do something, the flag should be '--dont-do-the-thing'. For internal variables, make it so it minimizes the number of negations you need so it reduces confusion.